### DISCUSSION TITLE
Possible library mislabeling?

### DISCUSSION THREAD

**Discussion by aspencoyle** [RobertsLab/resources#1351] [Q&A] [answered]
Hi all,

I was going through the FASTQ files in cbai_transcriptome_v2.0, available [here](https://gannet.fish.washington.edu/Atumefaciens/20200502_cbai_trinity_all_RNAseq/fastq.list.txt), and noticed that the library 329774 isn't present in the [list of pooled RNAseq samples](https://github.com/grace-ac/paper-tanner-crab/blob/master/data/sample-list_pool-RNAseq.csv). 

I also noticed that the inverse applies to 329771 (present in the formal list, not in the FASTQ files), so I assume this is a simple labeling error. But just in case, thought I'd check with @grace-ac and @kubu4!

**Comment by kubu4** [comment, marked as answer]
I'd make the same assumption, that it's a labelling error, as `329771` doesn't correspond to any library IDs in [Nightingales](https://b.link/nightingales).
